Update What's New to Include information on Timezone and Applicability changes in transformations (#76571)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v10-2.md

Learn more in the [Format string documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#format-string).

> ### Detect Unusable Transformations

Reviewer: ```suggestion
### Detect unusable transformations
```

<!-- Kyle Cunningham -->

> _Available behind a feature flag in Grafana Open Source and Enterprise_

Reviewer: We should talk about this in terms of a release stage. Does this align with [experimental, private preview, or public preview](https://grafana.com/docs/release-life-cycle/)?

> We've added initial support to detect situations in which various transformations won't work appropriately based on current data. Previously, in selecting transformations it would require a process of trial and error or otherwise knowing how a given transformation worked beforehand to configure them correctly. Transformations that we've detected can't be used will be shaded in the interface to indicate they can't be used along with a helpful message explaining why.

Reviewer: ```suggestion
We've added initial support to detect situations in which various transformations won't work appropriately based on current data. Previously, selecting the appropriate transformation and configuring it correctly required a process of trial and error or already knowing how a given transformation worked. Now, transformations that we've detected can't be used are shaded in the interface to indicate this, along with a helpful message explaining why.
```

{{< figure src="/media/docs/grafana/transformations/disabled-transformation.png" caption="Transformation that has been disabled because it doesn't have the necessary data" >}}

> For users with the `transformationsRedesign` feature flag set, you'll be able to access this functionality right away. For everyone who would like to try it, this feature flag can be enabled in your [Grafana configuration](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#feature_toggles).

Reviewer: ```suggestion
If you have the `transformationsRedesign` feature flag set, you'll be able to access this functionality right away. If you'd like to try it, enable this feature flag in your [Grafana configuration](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#feature_toggles).
```

### Extended time zone support in Format time and Convert field type transformations

<!-- Kyle Cunningham -->

_Generally available in Grafana Open Source and Enterprise_

> Support has been added to set timezones manually when formatting times as strings using the *Format time* and *Convert field type*. This allows times to be formatted relative to any timezone across the globe!

Reviewer: ```suggestion
We've added support for setting timezones manually when formatting times as strings using the **Format time** and **Convert field type** transformations. This allows times to be formatted relative to any timezone across the globe.
```

> {{< figure src="/media/docs/grafana/transformations/format-timezone.png" caption="Timezone support in the format time transformation" >}}

Reviewer: ```suggestion
{{< figure src="/media/docs/grafana/transformations/format-timezone.png" caption="Timezone support in the Format time transformation" >}}
```

> > > > > > > > docs/whats-new-v10.2

Reviewer: Not sure where this came from
```suggestion
```

Learn more in the [Format string documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#format-string).

### Detect unusable transformations

<!-- Kyle Cunningham -->

> _Available in public preview in all editions of Grafana_

Reviewer: I'll add this separately to the Cloud what's new and tag you for review.
```suggestion
_Available in public preview in all Grafana Open Source and Enterprise_
```

> _Available in public preview in all Grafana Open Source and Enterprise_

Reviewer: ```suggestion
_Available in public preview in Grafana Open Source and Enterprise_
```
